lib/wings.rb

```diff
@@ -92,16 +92,19 @@ module Wings; end
 custom_queries = [Hyrax::CustomQueries::Navigators::ChildCollectionsNavigator,
                   Hyrax::CustomQueries::Navigators::ChildFilesetsNavigator,
                   Hyrax::CustomQueries::Navigators::ChildWorksNavigator,
+                  Hyrax::CustomQueries::Navigators::FindFiles,
                   Wings::CustomQueries::FindAccessControl, # override Hyrax::CustomQueries::FindAccessControl
                   Wings::CustomQueries::FindFileMetadata, # override Hyrax::CustomQueries::FindFileMetadata
                   Wings::CustomQueries::FindManyByAlternateIds] # override Hyrax::CustomQueries::FindManyByAlternateIds
 custom_queries.each do |query_handler|
   Valkyrie.config.metadata_adapter.query_service.custom_queries.register_query_handler(query_handler)
 end
 
+Hyrax::Schema # rubocop:disable Lint/Void - required if a registered model using Hyrax::Schema to include core or basic metadata (e.g. PcdmFileSet)
```

Comment:
`rubocop:disable Lint/Void` needs to be re-enabled, yes? 

#4076 also resolves this issue. either reviewing that and rebasing, or simply cherry-picking that commit here could be a better solution.

Reply:
Glad you addressed this.  I wasn't happy with this solution.

Reply:
FYI... I didn't need to re-enable because it was inline.

This requires re-enabling...
```
# rubocop:disable Lint/Void
Hyrax::Schema
# rubocop:enable Lint/Void
```

This does not.  It only applies to this one line.
```
Hyrax::Schema # rubocop:disable Lint/Void
```

Reply:
oh huh. i had no idea! thanks for this!

Reply:
Removed Hyrax::Schema in the 3rd commit.